src/ekore/anomalous_dimensions/unpolarized/space_like/as4/fhmruvv_approximations/gnsm.py

```diff
@@ -0,0 +1,221 @@
+r"""The unpolarized, space-like anomalous dimension :math:`\gamma_{ns,-}^{(3)}`."""
+import numba as nb
+
+from eko.constants import CF, zeta3
+
+from ......harmonics import cache as c
+from ......harmonics.log_functions import lm11, lm11m1, lm12m1, lm13m1
+
+
+@nb.njit(cache=True)
+def gamma_nsm(n, nf, cache, variation):
+    r"""Compute the |N3LO| valence-like non-singlet anomalous dimension.
+
+    The routine is taken from :cite:`Moch:2017uml`.
+
+    The nf^{0,1} leading large-nc contributions and the nf^2 part are
```

Comment:
I'm perfectly happy with keeping the authors original comments, but can we make them verbatim somehow? (else they will render weirdly)